projectroles/forms.py

```diff
@@ -312,7 +312,14 @@ def _get_parent_choices(cls, instance, user):
         return sorted(ret, key=lambda x: x[1])
 
     def _set_app_setting_widget(self, app_name, s_field, s_key, s_val):
-        """Internal helper for setting app setting widget and value"""
+        """
+        Internal helper for setting app setting widget and value
+
+        :param app_name: App name
+        :param s_field: Form field name
+        :param s_key: Setting key
+        :param s_val: Setting value
+        """
```

Comment:
I appreciate writing out these docstrings in total. For internal helpers it's not necessarily critical, but does help understanding what's happening in development. I do try to write these out myself nowadays. Only one very tiny note, in case of a "full" docstring like this it is convention to add a period `.` after the description :)